src/MainWindow.cpp

```diff
@@ -318,6 +320,9 @@ void MainWindow::init()
         connect(ui->mainTab, &QTabWidget::tabCloseRequested, [=](int /*index*/) {
                 action->setChecked(ui->mainTab->indexOf(widget) != -1);
             });
+
+        if (widget == ui->pragmas)
+            viewMenuPragmas = action;
```

Comment:
Maybe is there a better solution to avoid having to store the action. Have you tried with findChild?
https://doc.qt.io/qt-5/qobject.html#findChild

Reply:
I have already tried it. However, `findChild` seems to search based on 'objectName', but it seems that 'objectName' is not specified in the existing code. `action->objectName()` return `0`.

Reply:
Then we can call [`setObjectName()`](https://doc.qt.io/qt-5/qobject.html#objectName-prop) using `widget->accessibleName()`